x-pack/test/siem_cypress/config.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I'm not entirely sure how this file made it into the backport. Looks like we can try to remove it?